src/flows/recommendation_api/shortreads_flow.py

```diff
@@ -0,0 +1,73 @@
+from typing import List
+from prefect import Flow, task, unmapped
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from api_clients.sqs import put_results, RecommendationCandidate, NewTabFeedID
+from common_tasks.corpus_candidate_set import (
+    create_corpus_candidate_set_record,
+    load_feature_record,
+    feature_group,
+    validate_candidate_items,
+)
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+FLOW_NAME = get_flow_name(__file__)
+
+CURATED_SHORTREADS_CANDIDATE_SET_ID_EN = "7ef90242-ff7a-44ac-8a32-53193e4a23eb"
+CURATED_SHORTREADS_CANDIDATE_SET_ID_DE = "57e4d3d1-9b4a-4a35-82f4-e577d88f6521"
+
+# Export approved corpus items by language and recency
+EXPORT_LONGREADS_ITEMS_SQL = """
+SELECT 
+    a.resolved_id as "ID", 
+    c.top_domain_name as "PUBLISHER"
+FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS" AS a
+JOIN "ANALYTICS"."DBT"."CONTENT" AS c
+  ON c.CONTENT_ID = a.CONTENT_ID
+WHERE a.REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD("day", -90, current_timestamp())
+AND c.WORD_COUNT <= 900
```

Comment:
Nit: I wouldn't bother doing this for the legacy SQS flows, but when we create the Corpus flow, I'd probably make the `word_count` a parameter, and merge these flows into one.

Reply:
the tricky part about that is that the inequality has to flip also.  I considered this, but it probably would require two parameters and some kind of flag to turn things off.